src/loaders/TextureLoader.js

```diff
@@ -19,6 +19,12 @@ Object.assign( THREE.TextureLoader.prototype, {
 		loader.setPath( this.path );
 		loader.load( url, function ( image ) {
 
+			// JPEGs can't have an alpha channel, so memory can be saved by storing them as RGB.
+			var isJPEG = url.search(/\.(jpg$)|(jpeg$)/) > 0;
```

Comment:
Could be written as `/\.(jpg|jpeg)$/`.

Will yield the same result, but should be (subjectively) cleaner and a bit more correct since it will look for `.jpeg` instead of `jpeg`